app/src/main/java/org/p2p/wallet/infrastructure/network/environment/NetworkServicesUrlProvider.kt

```diff
@@ -61,7 +61,7 @@ class NetworkServicesUrlProvider(
     fun loadNotificationServiceEnvironment(): NotificationServiceEnvironment {
         val url = sharedPreferences.getString(
             KEY_NOTIFICATION_SERVICE_BASE_URL,
-            context.getString(R.string.notificationServiceBaseUrl)
+            context.getString(R.string.notificationServiceTestBaseUrl)
```

Comment:
please leave todo or build app with test Url only for QA
i cant pass this change to release branch